test/src/util.js

```diff
@@ -35,16 +39,28 @@ export async function login() {
 /**
  * Register a new user with a random username.
  *
- * @returns {object} { username, email, password }
+ * @returns {object} { username, email, password, firstName, lastName }
  */
 export async function registerNewUser() {
   const username = `user${uniqueId()}`;
   const email = `mr${username}@dandi.test`;
   const password = 'XtR4-S3curi7y-p4sSw0rd'; // Top secret
 
+  // there's currently no way to set these in development mode,
+  // so they are always empty strings by default
+  const firstName = '';
+  const lastName = '';
```

Comment:
WE should think about a way to set these for testing purposes

Reply:
Filed an issue https://github.com/dandi/dandi-api/issues/495